### ISSUE TITLE
Extraction of authors and title from papers

### ISSUE THREAD

**Issue by gerking** [gi-ev/LNI-proceedings#12]
When generating the proceedings.tex, the authors' names are extracted from the square brackets `[...]` inside each paper.tex:

> Check for all paper.tex that all authors are the format \author[Firstname Lastname \and ...]{...}

This is somewhat unfortunate because authors often use these square brackets for abbreviations, e.g., `[Lastname et al.]`
However, if proceedings.tex uses these abbreviations, they will appear in the table of contents and proceedings.bib, where one would usually expect to see all authors.

Would it be possibly to extract the authors' names from the curly braces inside each paper.tex? The abbreviated forms could still be copied into proceedings.tex as the optional argument of `\addpaper`, so that they will be shown in the header of the respective paper.

(Obviously, the same applies to the paper titles.)

**Comment by koppor** [comment]
As far as I remember, I opted for `[]`, because the `{}` part contains references to institutions?

Example (source: https://github.com/gi-ev/LNI/blob/ea3d3e0916cbf930f99c76307789aa005a9e2343/lni-paper-example-de.tex#L46)

```latex
\author[Vorname1 Nachname1 \and Vorname2 Nachname2]
{Vorname1 Nachname1\footnote{Universität, Abteilung, Straße, Postleitzahl Ort, Land \email{emailaddress@author1}} \and
 Vorname2 Nachname2\footnote{University, Department, Address, Country \email{emailaddress@author2}}}
```

Someone with python skills could update https://github.com/gi-ev/LNI-proceedings/blob/1970749c7ca88cf0fc2e94a198e9d758071f86a2/addAuthTi.py#L25. Maybe with [LatexWalker](https://pylatexenc.readthedocs.io/en/latest/latexwalker/)? (See https://tex.stackexchange.com/a/570467/9075 for details)

**Comment by gerking** [comment]
> because the `{}` part contains references to institutions?
Yes, and sometimes also references to funding agencies etc. That's a problem of course, but the `[]` is often incomplete.
